apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -14,50 +52,243 @@
       :options="THEMES"
       @change="setTheme"
     />
-  </dt-stack>
-  <div v-for="category in categories" :key="category">
-    <slot :name="category">
-      <!-- slot for anchor header -->
-    </slot>
-    <token-table :category="category" :format="format" :theme="theme" />
+  </div>
+  <div v-for="category in Object.keys(filteredTokens)" :key="category">
```

Comment:
Seems like we are using `Object.keys(filteredTokens)` in many places.

Could we create a 

`const filteredTokensKeys = computed(() => Object.keys(filteredTokens.value));`

and replace them.